pom.xml

```diff
@@ -100,7 +100,7 @@
         <guava.version>13.0.1</guava.version>
         <gson.version>2.2.4</gson.version>
         <findbugs.jsr305.version>2.0.1</findbugs.jsr305.version>
-        <netty.version>4.1.16.Final</netty.version>
+        <netty.version>4.1.71.Final</netty.version>
```

Comment:
Can we bump this to `4.1.75.Final `to keep it consistent with the `netty-http` dependency?